indico/modules/events/registration/templates/display/regform_display.html

```diff
@@ -0,0 +1,10 @@
+{% block content %}
+    <div ng-app="nd" ng-controller="AppCtrl">
+        <div nd-reg-form
+            conf-id="{{ event.id }}"
+            conf-sections="{{ sections | tojson | forceescape }}"
+            conf-currency="{{ currency }}"
+            post-url="{{ url_for('.display_regform', regform) | tojson | forceescape }}"
+            csrf-token="{{ session.csrf_token }}"></div>
```

Comment:
Are you using this? You can always get it using `$('#csrf-token').attr('content');`